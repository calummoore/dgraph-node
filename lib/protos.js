'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; // @noflow


var _grpc = require('grpc');

var _grpc2 = _interopRequireDefault(_grpc);

var _path = require('path');

var _path2 = _interopRequireDefault(_path);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

const p1 = _grpc2.default.load(_path2.default.join(__dirname, '/../protos/api.proto'));
exports.default = _extends({}, p1.api);