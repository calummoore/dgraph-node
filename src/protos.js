// @noflow
import grpc from 'grpc'
import path from 'path'

const p1 = grpc.load(path.join(__dirname, '/../protos/api.proto'))
export default { ...p1.api }
