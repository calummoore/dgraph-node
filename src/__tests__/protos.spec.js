import protos from '../protos'

describe('protos', () => {
  it('match snapshot', () => {
    expect(protos).toMatchSnapshot()
  })
})
