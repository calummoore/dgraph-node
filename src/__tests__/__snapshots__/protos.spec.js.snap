// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`protos match snapshot 1`] = `
Object {
  "Assigned": [Function],
  "AssignedIds": [Function],
  "Check": [Function],
  "Dgraph": [Function],
  "Facet": [Function],
  "Latency": [Function],
  "LinRead": [Function],
  "Mutation": [Function],
  "NQuad": [Function],
  "Operation": [Function],
  "Payload": [Function],
  "Request": [Function],
  "Response": [Function],
  "SchemaNode": [Function],
  "TxnContext": [Function],
  "Value": [Function],
  "Version": [Function],
}
`;
